### PR TITLE
Bug 1122454 - Do not focus active app if the browser menu is opened

### PR DESCRIPTION
**Pull request by evanxd**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=10edcee46c9d)

**Comment by evanxd** [comment]
v 0.0.1: https://github.com/evanxd/gaia/commit/9f09bf9ccb824dcfdc50680c9e9b29f9bd1937eb

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=13f2b32177d5)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0e589cdc77b2)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=da8c3dc69f3b)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=13ae4e910a4e)

File: apps/system/js/app_window_manager.js [hunk 1]
```diff
@@ -57,7 +57,10 @@
         return;
       }
       this.debug('focusing ' + this._activeApp.name);
-      this._activeApp.focus();
+      var contextMenu = this._activeApp.contextmenu;
+      if (!(contextMenu && contextMenu.isShown())) {
+        this._activeApp.focus();
+      }
```
Comment (alivedise) [inline apps/system/js/app_window_manager.js]: Please do this in AppWindow#focus().

**Comment by evanxd** [comment]
v0.0.2: https://github.com/evanxd/gaia/commit/ba4cfebec5ad0cc9b7c6cfc76a41dcf2d95ef6df

File: apps/system/js/app_window_manager.js (hunk 1 above)
Comment (evanxd) [inline apps/system/js/app_window_manager.js]: Sure.

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dbbd80c1c562)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2e7db3f2b42c)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=502ecfcbb7fc)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f4ba1cf06ae1)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=68f5b5ea94be)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0fd241fa8589)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=38766a2fca84)

**Comment by try-server-hook** [comment]
[evanxd Evan Tseng (evanxd)](https://github.com/evanxd) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=053baa96ad04)

**Pull request merged**